### PR TITLE
default config fix

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do?

The PR fixes a problem in `oracle-dbm.d/conf.yaml.default`.

### Motivation

The problem was introduced with [the Oracle PR](https://github.com/DataDog/datadog-agent/pull/16449). Fortunately, @pgimalac [spotted the problem](https://dd.slack.com/archives/C011QG8MJBA/p1681831903327799) while doing configuration checks for the 7.45.0. release.

Also, the file was renamed from `conf.yaml.default` to `conf.yaml.example`, as suggested by @KSerrania [here](https://dd.slack.com/archives/CNY5XCHAA/p1681905686930419?thread_ts=1681835962.583579&cid=CNY5XCHAA) This is because the check should run only if explicitly enabled.

### Additional Notes

We should fix this in the release `7.45.0.

### Possible Drawbacks / Trade-offs

No drawbacks.

### Describe how to test/QA your changes

`datadog-agent checkconfig`

Ensure that there aren't any warnings at the beginning of the output of the command above.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by pgimalac — APPROVED** [review]
I can confirm that with this new default config the error is gone !

**Review by pgimalac — APPROVED** [review]

**Comment by pgimalac** [comment]
You probably want to update this line in CODEOWNERS file https://github.com/DataDog/datadog-agent/blob/5b275bc40d338eea37259780f6accf27a48e42e1/.github/CODEOWNERS#L121 so that the example file is owned by your team.

File: cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    ## Set to `true` to enable Database Monitoring.
+    #
+    # dbm: false
+
+    ## Configure collection of query metrics
+    #
+    # query_metrics:
+
+        ## @param enabled - boolean - optional - default: true
+        ## Enable collection of query metrics. Requires `dbm: true`.
```
Comment (jmeunier28) [inline cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example:78]: should the default for this be `true`?

File: cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+#
+init_config:
+
+  ## @param min_collection_interval - int - optional - default:10
+  ## Active session sampling interval in seconds
+  # min_collection_interval: <interval_in_seconds>
+
+## Every instance is scheduled independent of the others.
+#
+instances:
```
Comment (jmeunier28) [inline cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example:11]: which one of these is the config to actually enable activity collection?

Comment (nenadnoveljic) [reply cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example:11]: Activity sampling is enabled with setting `dbm: true`. Currently, there's no dedicated parameter for enabling/disabling activity sampling. I see no point in disabling activity samples, as every other DBM feature depends on it.

Comment (jmeunier28) [reply cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example:11]: sure, but the way all of our other integrations work is they allow for granular control over every single feature. Maybe this is fine for now, but when we add query metrics support fully we should allow for customers to turn on/off activity if they want

**Review by jmeunier28 — APPROVED** [review]

**Pull request merged**